docs/reference/errors-and-warnings/NU3024.md

```diff
@@ -0,0 +1,47 @@
+---
+title: NuGet Warning NU3024
+description: NU3024 Warning code
+author: heng-liu
+ms.author: henli
+ms.date: 3/18/2019
+ms.topic: reference
+ms.reviewer: dtivel
+f1_keywords: 
+  - NU3024
+---
+
+# NuGet Warning NU3024
+
+## Scenario 1
+
+<pre>Package 'SamplePackage v1.0.0' from source 'http://source.com/index.json': The primary signature's timestamp certificate has an unsupported digest algorithm.</pre>
+
+### Issue
+
+The certificate used to timestamp the package signature has an unsupported digest algorithm.
+
+
+### Solution
+
+Ensure that the timestamp authority's signing certificate has one of the following digest algorithms - 
```

Comment:
The message in the scenario sounds like a restore warning (so someone referencing the package), but the solution sounds like something only the package author can action. Is this correct?  If so, the solution should start with "Request the package author to ...", just like the second scenario below.

Reply:
Yes, you understanding is right. The first scenario is signing, so the solution is what the package author should do. The second scenario is validating when consuming a package, so the solution is what the package consumer should do. I think splitting them into two scenarios will be better. @dtivel What do you think?

Reply:
Sure, but the text in both scenarios and issues need to be updated after https://github.com/NuGet/NuGet.Client/pull/2741 is merged.

Reply:
NU3024 updated accordingly.